lib/build/amodro-trace/lib/loader/Loader.js

```diff
@@ -352,66 +355,77 @@ function __exec(contents, r, d, c) {
             layer.existingRequireUrl = url;
           }
         } else {
-          // Load the file contents, process for conditionals, then
-          // evaluate it.
-          contents = context.fileRead(moduleName, url);
+          traceCacheEntry = getOwn(context._traceCache, moduleName);
+          if (traceCacheEntry && falseProp(context.needFullExec, moduleName)) {
```

Comment:
So just as a note, this `falseProp` check should be the only difference between master amodrotrace and our take on it.